### PR TITLE
Updates module

### PR DESCRIPTION
**Pull request by richese**
A generic update checker.
This is basically just stripped down mail module with different backends.

Current backends are:
* Pacman - Arch. Is using the checkupdates script.
* Cower - Arch. For AUR packages.
* AptGet - Debian. It's not ideal but it seems to work as intended. I only tested this on fresh install of Debian 8.

### CODE REVIEW

**Comment by facetoe** [comment]
This is awesome. I've tested the `Pacman` and `Cower` backends and they both work perfectly on my machine.

One thing that would be nice though is to be able to see how many updates are pending for each backend. Maybe a `name` variable could be added to the backend and an option could be provided to show the label before each count. Or maybe you could scoll through total, and then each backend. 

Anyway nice work!

**Comment by richese** [comment]
I would rather solve that by using multiple modules in config file to keep the code simple. Usage of multiple backends is Arch Linux specific thing, since repos and AUR are separated.

File: i3pystatus/updates/__init__.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+        status.run()
+
+    """
+
+    interval = 3600
+
+    settings = (
+        ("backends", "Required list of backends used to check for updates."),
+        ("format", "String shown when updates are availible. "
```
Comment (enkore) [inline i3pystatus/updates/__init__.py]: "available" instead of "availible"

**Comment by enkore** [comment]
> This is awesome. 

Indeed very nice work!

**Comment by facetoe** [comment]
@richese Ahhh good point about the multiple backends being Arch specific. Mutliple instances it is!

**Pull request merged**